core/sis-referencing/src/test/java/org/apache/sis/geometry/CoordinateFormatTest.java

```diff
@@ -188,4 +193,76 @@ public void testParseGeographic4D() throws ParseException {
                          e.getLocalizedMessage());  // In the language specified at CoordinateFormat construction time.
         }
     }
+
+    @Test
+    public void testCreatesCoordinateFormatTaking2ArgumentsAndCallsClone() {
+        Locale locale = Locale.CANADA;
+        CoordinateFormat coordinateFormat = new CoordinateFormat(locale, null);
+        CoordinateFormat coordinateFormatTwo = coordinateFormat.clone();
+
+        assertNotSame(coordinateFormatTwo, coordinateFormat);
+    }
+
+    @Test
+    public void testParseAndFormatTaking2Arguments() throws IOException, ParseException {
+        CoordinateFormat coordinateFormat = new CoordinateFormat();
+        DirectPosition2D directPosition2D = new DirectPosition2D();
```

Comment:
Could be set to a test position different than (0, 0) for better chances to catch errors.